Sources/Testing/Events/Event.swift

```diff
@@ -447,3 +448,36 @@ extension Event.Kind {
     }
   }
 }
+
+extension Event.Context {
+
+  /// A serializable type which provides context about a posted ``Event``.
+  ///
+  @_spi(ForToolsIntegrationOnly)
+  public struct Snapshot: Sendable, Codable {
```

Comment:
Did you mean to adopt this in the experimental JSON output code too?

Reply:
Not initially but I just did :)